llamacpp-server-launcher.py

```
@@ -1311,6 +1384,20 @@ def _setup_chat_template_tab(self, parent):
 
         ttk.Separator(frame, orient='horizontal').grid(column=0, row=r, columnspan=3, sticky='ew', padx=5, pady=10); r += 1
 
+        # --- Jinja rendering toggle ---
+        # Controls whether --jinja is passed to the server so Jinja chat templates render.
+        self.jinja_enabled_check = ttk.Checkbutton(
+            frame,
+            variable=self.jinja_enabled,
+            text="Enable Jinja template rendering (--jinja)",
+        )
+        self.jinja_enabled_check.grid(column=0, row=r, columnspan=3, sticky="w", padx=5, pady=(0, 3)); r += 1
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)